swatch-core/src/main/resources/swatch-core/application.yaml

```diff
@@ -8,24 +8,8 @@
 # result in an exception at start-up when the clowder property cannot be successfully resolved.
 # The values can still be overridden using environment variables as they have a higher precedence
 # than the defaults defined here.
-KAFKA_USERNAME: ${clowder.kafka.brokers[0].sasl.username:}
-KAFKA_PASSWORD: ${clowder.kafka.brokers[0].sasl.password:}
-KAFKA_SASL_PROTOCOL: ${clowder.kafka.brokers[0].sasl.securityProtocol:}
-KAFKA_SASL_MECHANISM: ${clowder.kafka.brokers[0].sasl.saslMechanism:}
 DATABASE_SSL_MODE: ${clowder.database.sslMode:disable}
 DATABASE_SSL_CERT: ${clowder.database.rdsCa:}
-
-# This needs to be a PEM encoded certificate string.  If you want to point to a file you can use
-# spring.kafka.ssl.trust-store-location property but that property is mutually exclusive with the
-# spring.kafka.ssl.trust-store-certificates property.  Since Clowder gives us the actual
-# certificate PEM, we have to use the trust-store-certificates property here.
-#
-# The value of this property must match a regular expression defined in Kafka's
-# DefaultSslEngineFactory.PemParser class.  The regular expression is (with Java escapes)
-# -+BEGIN\\s*.*CERTIFICATE[^-]*-+\\s+(?:\\s*[^\\r\\n]*:[^\\r\\n]*[\\r\\n]+)*([a-zA-Z0-9/+=\\s]*)-+END\\s*.*CERTIFICATE[^-]*-+\\s+")
-# A notable point about this expression is that the "----BEGIN CERTIFICATE-----" piece must be
-# followed by a newline which makes it challenging to place the value in a properties or YAML file.
```

Comment:
I'd like to retain this comment down below in the `kafka.properties.ssl` YAML section since it's still relevant.

Reply:
Re-added it to the relevant property in spring.kafka.properties.ssl.trust-store-certificates